### PR TITLE
Fixes the build of the stress test project

### PR DESCRIPTION
**Pull request by DavidObando**
@JunTaoLuo @muratg @SajayAntony 

Adding packages that unblock building stress-test under the CLI.

### CODE REVIEW

File: tools/LargePageGenerator/project.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-*",
-    "System.Xml.XDocument": "4.0.11-rc2-23706"
+    "System.Xml.XDocument": "4.0.11-rc2-23722",
+    "System.Linq": "4.0.1-rc2-23722"
```
Comment (JunTaoLuo) [inline tools/LargePageGenerator/project.json]: Is there a reason these are pinned?

**Comment by pranavkm** [comment]
cc @JunTaoLuo. He's a lot more familiar with this repo

File: tools/LargePageGenerator/project.json (hunk 1 above)
Comment (DavidObando) [inline tools/LargePageGenerator/project.json]: I couldn't find a reason for it. I can star it. Any input @pakrym?

**Comment by JunTaoLuo** [comment]
LGTM

File: tools/LargePageGenerator/project.json (hunk 1 above)
Comment (DavidObando) [inline tools/LargePageGenerator/project.json]: Starring it works well, will send the change over to this PR.

**Pull request merged**